website/docs/providers/ldap/general_setup.md

```diff
@@ -0,0 +1,84 @@
+---
+title: General Setup
+---
+
+### Create User/Group
+
+1. Create a new user account to test ldap bind. `Directory > Users > Create` In this example `ldapservice`.
+   Note the DN is `cn=ldapservice,ou=users,dc=ldap,dc=goauthentik,dc=io`
+2. Create a new group for ldap searches. In this example `ldapsearch`. Add the `ldapservice` user to this new group.
+
+:::info
+Note: The `default-authentication-flow` can have mult-factor authenticator(s) configured. This will not work with ldap for authentication. To account for this, create a new authenication flow, with 3 stages, dedicated to ldap.
+:::
+
+### Create Custom Stages
+
+1. Create a new identification stage. `Flows & Stage > Stages > Create`
+   ![](./general_setup1.png)
+2. Name it something meaningful like `ldap-identification-stage`. Select User fields Username and Email (and UPN if it is relevant to your setup).
+   ![](./general_setup2.png)
+3. Create a new password stage. `Flows & Stage > Stages > Create`
+   ![](./general_setup3.png)
+4. Name it something meaningful like `ldap-authentication-password`. Leave the defaults for Backends.
+   ![](./general_setup4.png)
+5. Create a new user login stage. `Flows & Stage > Stages > Create`
+   ![](./general_setup5.png)
+6. Name it something meaningful like `ldap-authentication-login`.
+   ![](./general_setup6.png)
+
+### Create Custom Flow
+
+1. Create a new authentication flow. `Flows & Stage > Flows > Create`. Name it something meaningful like `ldap-authentication-flow`
+   ![](./general_setup7.png)
+2. Click the newly created flow and choose `Stage Bindings`.
+   ![](./general_setup8.png)
+3. Click `Bind Stage` choose `ldap-identification-stage` and set the order to `10`.
+   ![](./general_setup9.png)
+4. Click `Bind Stage` choose `ldap-authentication-password` and set the order to `20`.
```

Comment:
Since the password stage is used with the identification stage, it shouldn't be directly bound to the flow

Reply:
This was how I have it setup and working. Are you saying if I remove the password stage from the flow and only use it in the identification stage, it will work?

Reply:
I figured it out and corrected